components/Banner/src/MDCBannerView.m

```diff
@@ -60,9 +60,9 @@ @interface MDCBannerView ()
 // Buttons constraints
 @property(nonatomic, readwrite, strong) NSLayoutConstraint *buttonContainerConstraintLeading;
 @property(nonatomic, readwrite, strong)
-    NSLayoutConstraint *buttonContainerConstraintLeadingWithTextLabel;
+    NSLayoutConstraint *buttonContainerConstraintWidthWithLeadingButton;
 @property(nonatomic, readwrite, strong)
-    NSLayoutConstraint *buttonContainerConstraintLeadingWithTextLabelGreater;
+    NSLayoutConstraint *buttonContainerConstraintLeadingWithTextLabel;
```

Comment:
Please add docs so it's easier to understand what this property does.

Reply:
Addressed in 6e7588f8128db5c72fc411fea383a46b607b31ad.